app/labor/flare_tag.rb

```diff
@@ -20,7 +20,7 @@ def initialize(article, except_tag = nil)
 
   def tag
     @tag ||= Rails.cache.fetch("article_flare_tag-#{article.id}-#{article.updated_at}", expires_in: 12.hours) do
-      flare = FLARE_TAGS.detect { |tag| article.cached_tag_list_array.include?(tag) }
+      flare = FLARE_TAGS.detect { |tag| article.cached_tag_list_array.include?(tag) } # Take the first flare tag to show up in the array
```

Comment:
since the line is already quite long, I'd put the comment on the line before instead of at the end